src/main/java/uk/gov/hmcts/ccd/data/definition/DefaultCaseDefinitionRepository.java

```diff
@@ -172,31 +181,39 @@ public CaseTypeDefinitionVersion doGetLatestVersion(String caseTypeId) {
     }
 
     @Override
-    @Cacheable("caseTypeDefinitionsCache")
-    public CaseType getCaseType(int version, String caseTypeId) {
-        return this.getCaseType(caseTypeId);
+    public List<Jurisdiction> getJurisdictions(List<String> ids) {
+        ArrayList<Jurisdiction> jurisdictions = new ArrayList<>(ids.size());
+        ids.forEach(id -> jurisdictions.add(getJurisdiction(id)));
+        return jurisdictions;
     }
 
+    @Cacheable(value = "jurisdictionListCache")
```

Comment:
shall we rename `jurisdictionListCache` to `jurisdictionCache`?

Reply:
Already was done. :)